src/pages/index.tsx

```diff
@@ -1,30 +1,166 @@
 
-function HomepageHeader() {
-  const {siteConfig} = useDocusaurusContext();
+function RotatingHeader() {
+  const { siteConfig } = useDocusaurusContext();
+  const actions = ['Build faster with', 'Integrate easier on', 'Cache data with', 'Publish messages using'];
+  const [index, setIndex] = useState(0);
+  const [fade, setFade] = useState(true);
+
+  useEffect(() => {
+    const interval = setInterval(() => {
+      setFade(false);
+      setTimeout(() => {
+        setIndex((prevIndex) => (prevIndex + 1) % actions.length);
+        setFade(true);
+      }, 500);
+    }, 5000);
+    return () => clearInterval(interval);
+  }, []);
+
   return (
-    <header className={clsx('hero hero--primary', styles.heroBanner)}>
-      <div className="container">
-        <h1 className="hero__title">{siteConfig.title}</h1>
-        <p className="hero__subtitle">{siteConfig.tagline}</p>
-      </div>
-    </header>
+    <div className={styles.headerSection}>
+      <h1 className={`${styles.rotatingHeader} ${fade ? styles.fadeIn : styles.fadeOut} ${styles.header}`}>
+        {actions[index]}
+      </h1>
+      <h1 className={styles.header}>the Momento Platform</h1>
+      <p className={styles.headerDescription}>Welcome to the Momento developer docs. Discover all the resources you need to help you get the most out of our products.</p>
+    </div>
   );
 }
 
 export default function Home(): JSX.Element {
   return (
     <Layout
       description="Description will go into a meta tag in <head />">
-      <HomepageHeader />
-      <main>
-        <HomepageFeatures />
+      <main className={styles.content}>
+        <div className={styles.contentContainer}>
+          <RotatingHeader />
+          <div className={styles.desktopOnly}>
+            <div className={styles.columns}>
+              <div>
+                <h2>Cache</h2>
+                <ul className={styles.columnList}>
+                  <li><a href="/cache/getting-started">Getting started</a></li>
+                  <li><a href="/cache/develop/api-reference">API reference</a></li>
+                  <li><a href="/cache/develop/guides/working-with-files-cache">How to cache files</a></li>
+                  <li><a href="/cache/how-to/rate-limiter">Build a rate-limiter</a></li>
+                </ul>
+              </div>
+              <div>
+                <h2>Topics</h2>
+                <ul className={styles.columnList}>
+                  <li><a href="/topics">About our serverless event bus</a></li>
+                  <li><a href="/cache/develop/guides/working-with-files-cache">Integrate with Amazon EventBridge</a></li>
+                  <li><a href="/topics/webhooks">Webhooks overview</a></li>
+                  <li>
+                    <a href="https://www.gomomento.com/blog/how-to-create-a-slack-workflow-with-webhooks-in-momento-topics" target='_blank'>
+                      <div className={styles.openInNew}>
+                        <span>Publish messages to Slack</span>
```

Comment:
Curious how you're selecting a sub-content for the landing page. Like I would have added the `running async tasks` pattern here, and I am not contesting your decision on this one, but curious about the reasoning.

Reply:
I know we spoke about this in Slack, but repeating here for posterity: I chose things that I felt had the highest likelihood of someone clicking on. I'm trying to pique their curiosity as best I can to get them to stay on the site as long as possible.